index.js

```diff
@@ -23,85 +24,97 @@ function Redis (host, port, pass, db = 0) {
 // Public Methods
 // TODO: check if this.ready, else reconnect and/or queue?
 
+function does_callback_exist(callback) {
+  return callback || function () {};
+}
+
+function handle_err_log(err) {
+  if (err) {
+    R5.out.error(err);
+  }
+}
+
 Redis.prototype.connect = function () {
-  if (this.ready) { return; }
+  if (this.ready) {
+    return;
+  }
 
   this.client = require('redis').createClient({
     host: this.host,
     port: this.port,
     password: this.pass,
-    db: this.db
+    db: this.db,
   });
 
-  let _this = this;
+  const _this = this;
   this.client.on('ready', function () {
     R5.out.log(`Connected to Redis (db: ${this.db})`);
     _this.ready = true;
   });
 
-  this.client.on('error', function (err) {
+  this.client.on('error', (err) => {
     R5.out.error(`Redis error: ${err}`);
     _this.ready = false;
     _this.connect();
   });
 };
 
-Redis.prototype.get = function (key, callback) {
-  if (!callback) { callback = function () { }; }
-
-  this.client.get(key, function (err, data) {
-    if (err) { R5.out.error(err); }
+Redis.prototype.handle_client_oper_action = function (action, key, callback) {
+  does_callback_exist(callback);
+  this.client[action](key, (err, data) => {
+    handle_err_log(err);
     callback(err, data);
   });
 };
 
+Redis.prototype.get = function (key, callback) {
+  this.handle_client_oper_action('get', key, callback);
+};
+
 Redis.prototype.set = function (key, value, key_expiration, callback) {
-  if (!callback) { callback = function () { }; }
+  does_callback_exist(callback);
 
   if (typeof value === 'object') {
-    R5.out.log('Passing a string into set is recommended (currently passed in object)');
+    R5.out.log(
+      'Passing a string into set is recommended (currently passed in object)',
+    );
     value = stringify(value);
   }
 
   if (key_expiration) {
     if (value) {
       this.client.set(key, value, 'EX', key_expiration, handle_data);
-    }
-    else {
+    } else {
       this.client.setex(key, key_expiration, 'EX', handle_data);
     }
-  }
-  else {
+  } else {
     this.client.set(key, value, handle_data);
   }
 
-  function handle_data (err, data) {
-    if (err) { R5.out.error(err); }
+  function handle_data(err, data) {
```

Comment:
we need to have all spaces between all of these.. basically, the rule is to have a space when defining the function and no space when calling the function. E.g., 
```
function handle_data (err, data) { ... }
...
handle_data('an-error', 'some-data');
```